django/utils/dateformat.py

```diff
@@ -325,8 +325,8 @@ def W(self):
         return self.data.isocalendar()[1]
 
     def y(self):
-        "Year, 2 digits; e.g. '99'"
-        return str(self.data.year)[2:]
+        "Year, 2 digits; e.g. '99', with leading zero"
```

Comment:
```suggestion
        "Year, 2 digits; e.g. '99', with leading zeros."
```
We should also update `date` [docs](https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#date) for the `y` format add a small `versionchanged` annotations below the table with formats.